### PR TITLE
Feat: Limit Orders Book

### PR DESCRIPTION
**Pull request by dinhtienn**

### CODE REVIEW

File: src/components/swapv2/LimitOrder/ListLimitOrder/TabSelector.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  return (
+    <Flex
+      sx={{
+        borderTopLeftRadius: upToSmall ? 0 : '19px',
+        overflow: 'hidden',
+        width: upToSmall ? '100%' : 'fit-content',
+      }}
+    >
+      <TabButton
+        style={style}
```
Comment (viet-nv) [inline src/components/swapv2/LimitOrder/ListLimitOrder/TabSelector.tsx:33]: dont use inline-style here. If you need to customize TabButton, just add `className` props into that component then re-style it here. like this:
```
const StyledTabButton = styled(TabButton)`
....
`
```

File: src/components/swapv2/LimitOrder/OrderBook/OrderItem.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  amount,
+  currency,
+  upToSmall,
+}: {
+  plus?: boolean
+  amount: string
+  currency?: Currency
+  upToSmall?: boolean
+}) => (
+  <Text>
```
Comment (viet-nv) [inline src/components/swapv2/LimitOrder/OrderBook/OrderItem.tsx:41]: do we need this Text?

File: src/components/swapv2/LimitOrder/OrderBook/OrderItem.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  const theme = useTheme()
+  const upToSmall = useMedia(`(max-width: ${MEDIA_WIDTHS.upToSmall}px)`)
+  const { currencyIn, currencyOut } = useLimitState()
+
+  return (
+    <ItemWrapper style={style}>
+      <Rate reverse={reverse}>{order.rate}</Rate>
+      <AmountInfo plus={reverse} amount={order.firstAmount} currency={currencyIn} upToSmall={upToSmall} />
+      <AmountInfo plus={!reverse} amount={order.secondAmount} currency={currencyOut} upToSmall={upToSmall} />
+      <Text style={{ color: rgba(theme.white, 0.6) }}>
```
Comment (viet-nv) [inline src/components/swapv2/LimitOrder/OrderBook/OrderItem.tsx:68]: how about color={theme.subText}

File: src/components/swapv2/LimitOrder/OrderBook/TableHeader.tsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+import { Text } from 'rebass'
+import styled from 'styled-components'
+
+import { useLimitState } from 'state/limit/hooks'
+import { MEDIA_WIDTHS } from 'theme'
+
+import { ItemWrapper } from './OrderItem'
+
+const Header = styled(ItemWrapper)`
+  background: ${({ theme }) => rgba(theme.white, 0.04)};
```
Comment (viet-nv) [inline src/components/swapv2/LimitOrder/OrderBook/TableHeader.tsx:13]: is theme.tableHeader suitable here?

**Review by viet-nv — CHANGES_REQUESTED** [review]

File: src/components/swapv2/LimitOrder/OrderBook/OrderItem.tsx (hunk 2 above)
Comment (dinhtienn) [reply src/components/swapv2/LimitOrder/OrderBook/OrderItem.tsx:41]: No, we don't @viet-nv

File: src/components/swapv2/LimitOrder/OrderBook/OrderItem.tsx (hunk 3 above)
Comment (dinhtienn) [reply src/components/swapv2/LimitOrder/OrderBook/OrderItem.tsx:68]: `color={theme.subText}` is ok

File: src/components/swapv2/LimitOrder/ListLimitOrder/TabSelector.tsx (hunk 1 above)
Comment (viet-nv) [reply src/components/swapv2/LimitOrder/ListLimitOrder/TabSelector.tsx:39]: is this updated @dinhtienn? I see you resolved this but I didn't see any change

Comment (dinhtienn) [reply src/components/swapv2/LimitOrder/ListLimitOrder/TabSelector.tsx:39]: @viet-nv Sorry It's a misunderstanding

**Review by viet-nv — APPROVED** [review]

**Pull request merged**